recipes-wpe/wpeframework/include/firmwarecontrol.inc

```diff
@@ -0,0 +1,21 @@
+WPE_FIRMWARECONTROL_EXTRAFLAGS += "${@bb.utils.contains('PREFERRED_PROVIDER_virtual/mfrlib', 'mfrlibs-rpi', '-DPLUGIN_FIRMWARECONTROL_PLATFORM_RPI=ON', '', d)}"
+
+# Provider for RPI platform has to be updated once the implementation for mfr fwupgrade is available
+RDEPENDS_${PN} += "${@bb.utils.contains('PREFERRED_PROVIDER_virtual/mfrlib', 'mfrlibs-rpi', 'mfrlibs-rpi', '', d)}"
+RDEPENDS_${PN} += "${@bb.utils.contains('PREFERRED_PROVIDER_virtual/mfrfwlib', 'fwupgrade-lib', 'fwupgrade-lib', '', d)}"
```

Comment:
Are these runtime dependencies? or Build dependencies?

Reply:
Yes, these are build and runtime dependent modules